modules/selenium/pom.xml

```diff
@@ -33,5 +33,26 @@
             <artifactId>jetty</artifactId>
             <version>6.1.25</version>
         </dependency>
+
+        <!-- Concise and flexible API for polling hub container, until it's up and running -->
```

Comment:
forgot to change a comment... this lib is used for all containers polling, not only for hub (see overridden `waitUntilContainerStarted`)